tests/general/InvalidLinkBearTest.py

```diff
@@ -126,6 +126,7 @@ def test_run(self):
         http://httpbin.org/status/410
         http://httpbin.org/status/500
         http://httpbin.org/status/503
+        http://httpbin.org/status/200 http://httpbin.org/status/404
```

Comment:
To test properly the second url in the line needs to be a new invalid link,
And that will mean the test will fail until you add the new link in the expected results below.